tests/conftest.py

```diff
@@ -101,12 +121,22 @@ def cookiecutter_repository(cookiecutter_project: Path) -> git.Repository:
 def cookiecutter_instance(
     cookiecutter_repository: git.Repository,
     context: Dict[str, str],
-    tmp_path: Path,
+    flavor_path: Path,
+    flavor: str,
 ) -> Path:
     """The cookiecutter instance."""
     template = str(cookiecutter_repository.path)
-    cookiecutter(template, no_input=True, output_dir=str(tmp_path))
-    return tmp_path / context["project_slug"]
+    if flavor == "vanilla":
+        cookiecutter(template, no_input=True, output_dir=str(flavor_path))
+        return flavor_path / context["project_slug"]
+    elif flavor == "cruft":
+        import cruft  # type: ignore[import]
```

Comment:
- https://github.com/cruft/cruft/pull/262

Reply:
```suggestion
        import cruft
```
They merged it, now waiting for the release.